scanner/src/routes/+page.svelte

```diff
@@ -181,16 +190,49 @@
 
     return true
   }
+
 </script>
 
 <main>
-  <div id="reader" />
-  {#if !scanning}
-    <div class="controls">
-      <button on:click={startScan}>Start scanning</button>
-      <button on:click={scanPreset}>Simulate scan of preset data</button>
+  <div id="reader">
+    {#if !scanning}
+      <div class="background" />
+    {/if}
+    <div class="relative barcode-scanner-area-wrapper">
+      <div class="relative barcode-scanner-area">
+        <img class="subtract-image" src={subtract} alt="subtract "/>
+        {#if scanning}
+          <div class="square surround-cover" />
+        {:else}
+          <img class="logo" src={logo} alt="logo" />
+        {/if}
+      </div>
+      <div class="relative">
+        {#if scanning}
+          <p class="description">Scan a Product Passport QR code to view product data</p>
+        {:else}
+          <p class="description">Turn on your camera for scan a Product Passport QR code to view product data</p>
+        {/if}
+      </div>
+      {#if !scanning}
+        <div class="relative button-wrapper">
+          <Button onClick={startScan} icon={camera} title="Turn on" />
+        </div>
+      {/if}
     </div>
-  {/if}
+    {#if scanning}
+      <div class="relative example-code-wrapper">
+        <div class="example-description">Here's an example to identify an <strong>IOXIO Tag</strong></div>
+        <div class="example-code">
+          <img src={qrcode} alt="code" />
+        </div>
+      </div>
+    {/if}
```

Comment:
I would prefer us having a route per screen, as it will be a lot easier to manage state and jump to different screens like that.